spire-flex/templates/agent-daemonset.yaml

```diff
@@ -15,6 +15,9 @@ spec:
       containers:
         - name: {{ $.Release.Name }}-agent
           image: {{ include "spire-flex.agent.image" . }}
+          args:
+            - -config
+            - "{{ default "/etc/spire" ((.Values.agent).configVolumeMountPath) }}/agent_conf"
```

Comment:
I think #15 covers this in more depth, with the ability to both set the path and the config file name as one variable **.agent.configFile**.  In addition, it comes with unit tests, a schema update, and documentation.

While I hate to discourage the "first MR" I think we just didn't coordinate well enough on this point to not step on each other's efforts.  Please review #15 and then decide if this effort should be withdrawn.

Now, I'll readily admit that in #15 I didn't know what a good default would be for the config file, so feel free to suggest better than I supplied.  I'm thinking "close to the spire examples / default spire install" but often that comes with a revision number in the path "spire-1.8.0", and it's far from LSB or any other standard, so I'm open to ideas.